ietf/doc/views_doc.py

```diff
@@ -1216,6 +1216,8 @@ def document_ballot_content(request, doc, ballot_id, editable=True):
         else:
             position_groups.append(g)
 
+    iesg = doc.get_state("draft-iesg")
```

Comment:
There is a get_state_slug function that would have done the same as these two lines